neqo-transport/src/server.rs

```diff
@@ -578,6 +588,18 @@ impl Server {
                 packet.wire_version(),
                 self.conn_params.get_versions().all(),
             );
+
+            crate::qlog::server_version_information_failed(
+                &mut self.create_qlog_trace(&AttemptKey {
+                    remote_address: dgram.source(),
+                    odcid: packet.dcid().into(),
+                }),
+                self.conn_params.get_versions().all(),
+                packet
+                    .version()
+                    .expect("version on `OtherVersion` and `Initial` to be set"),
```

Comment:
Same as `unwrap` in line 567 / 573. Let me know if you would prefer making this state impossible at compile time instead.

Reply:
:facepalm: not the same. Will fix.